client/me/purchases/paths.js

```diff
@@ -2,6 +2,10 @@ function list() {
 	return '/purchases';
 }
 
+function listNotice( noticeType = ':noticeType?' ) {
```

Comment:
That is an oversight, as originally, I tried to add this behavior to the `list` function. I'll update it now.